cmd/ipfs/main.go

```diff
@@ -500,13 +501,13 @@ func (i *cmdInvocation) setupInterruptHandler() {
 			n, err := ctx.GetNode()
 			if err != nil {
 				log.Error(err)
-				fmt.Println("Received interrupt signal, terminating...")
```

Comment:
Didn't find any good reason why this used the word 'terminating' instead of 'shutting down'